cmd/main.go

```diff
@@ -121,19 +124,47 @@ func serverSetup(options *Options, logger *zap.Logger) {
 		logger.Fatal("Failed in initialize s3 COS driver", zap.Error(err))
 		os.Exit(1)
 	}
-	serveMetrics(options.MetricsAddress, logger)
+	serveMetrics(options.ServerMode, options.MetricsAddress, logger)
 	S3CSIDriver.Run()
 }
 
-func serveMetrics(metricsAddress string, logger *zap.Logger) {
-	logger.Info("starting metrics endpoint")
+func serveMetrics(mode, metricsAddress string, logger *zap.Logger) {
+	logger.Info("starting metrics & cos-csi-mounter socket-health endpoints")
```

Comment:
Just noticed that log message will be printed for both controller server and node server. Hence, lets log message depending on whether its controller server and node server.

```
logMsg := "starting metrics endpoint"
if strings.Contains(mode, "node") {
  logMsg := "starting metrics & cos-csi-mounter socket-health endpoints"
}
logger.Info(logMsg)
```